includes/components/class-editor.php

```diff
@@ -373,6 +373,7 @@ public function __call( $name, $args ) {
 
 			// Get block arguments.
 			$block_type = substr( $name, strlen( 'render_' ) );
+			$block_name = substr( $name, strlen( 'render_' ) );
```

Comment:
You probably can't assume that the block name equals to the block type, sometimes there's a difference (e.g. related_listings block name, listings block type), so a different solution is required (maybe check in block_args? if missing, add on the rendered block side, in Bookings).